### PR TITLE
Adding BlockedAnonymousJoinClient to Teams meeting policy (New/Set)

### PR DESCRIPTION
**Pull request by tomkau**

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]
Approved pending @get-itips change.

**Comment by yogkumgit** [comment]
@tomkau Please review and implement the feedback. Thanks

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -129,6 +130,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -BlockedAnonymousJoinClientTypes
+A user can join a Teams meeting anonymously using a [Teams client](https://support.microsoft.com/office/join-a-meeting-without-a-teams-account-c6efc38f-4e03-4e79-b28f-e65a4c039508) or using a [custom application built using Azure Communication Services](/azure/communication-services/concepts/join-teams-meeting.md). When anonymous meeting join is enabled, both types of clients may be used by default. This optional parameter can be used to block one of the client types that can be used.
+
+The Allows values are ACS (to block the use of Azure Communication Services clients) or Teams (to block the use of Teams clients). Both can also be specified, separated by a comma, but this is equivalent to disabling anonymous join completely.
```
Comment (get-itips) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:136]:
```suggestion
The allowed values are ACS (to block the use of Azure Communication Services clients) or Teams (to block the use of Teams clients). Both can also be specified, separated by a comma, but this is equivalent to disabling anonymous join completely.
```

**Comment by get-itips** [comment]
@tomkau sorry, one more edit

**Review by get-itips — APPROVED** [review]

**Pull request merged**